src/constants/MarkdownFiles/posts/2025-06-30-gsoc-25-AdityaKrSingh26-week07.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "GSoC ’25 Week 07 Update by Aditya Kumar Singh"
+excerpt: "Enhanced shared mode synchronization for Tour and Doctor activities, improved scoring visualization, and camera state persistence in the 3D Human Activity."
+category: "DEVELOPER NEWS"
+date: "2025-06-30
```

Comment:
Please fix the date by closing the quotations.

Otherwise, this recent post ends up at the end of the list for http://localhost:5173/news/all